src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  border: 1px solid $border-main-color;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  animation-duration: $animation-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 32 auto 40px;
+  }
+
+  &__info {
+    width: 166px;
+    margin: 0 auto 16px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 auto 4px;
+
+    width: 166px;
+    height: 36px;
+
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    margin: 0;
+
+    color: #616070;
+  }
+
+  &__price {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    margin: 0 auto;
+  }
+
+  &__price-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: #616070;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    width: 166px;
```

Comment:
Don't fix container size (if there is no such a requirement)

Fix in all places where it's not necessary